src/qmflows/cp2k_utils.py

```diff
@@ -243,16 +248,16 @@ def set_prm(settings: Settings, key: Union[str, Tuple[str, ...]],
     :data:`CP2K_KEYS_ALIAS` : :class:`dict` [:class:`str`, :class:`tuple` [:class:`str`, ...]]
         A dictionary mapping ``key_path`` aliases to the actual keys.
 
-    """  # noqa
+    """  # noqa: E501
     if isinstance(value, abc.Sequence):
-        for prm_map in value:
+        for prm_map in value:  # type: Union[MappingScalar, MappingSequence]
```

Comment:
A type comment. In this example it is used for assigning a type to the elements of a sequences (*i.e.* ``prm_map``).

Reply:
I think it is good for documentation purposes